README.md

```diff
@@ -50,3 +51,25 @@ terraform plan
 popd
 ```
 
+## Running Tests
+
+To run the unit tests:
+
+```bash
+make test
+```
+
+If you want to run the tests multiple times without having to spin the backend
+up and down for every run:
+
+```bash
+# Start the required dependencies and load env variables
+make up
+declare -x $(cat env/local.env)
```

Comment:
The `declare` command is shell-specific and should be noted.  If you do back out the `up` change, I'll add these instructions in my PR.